Source/iOS/UIImage+Hue.swift

```diff
@@ -146,4 +146,22 @@ extension UIImage {
       secondaryColor ?? (isDarkBackgound ? whiteColor : blackColor),
       detailColor    ?? (isDarkBackgound ? whiteColor : blackColor))
   }
+
+  public subscript (x: Int, y: Int) -> UIColor? {
```

Comment:
I don't know, but I think I agree that function is better for autocompletion. Will change a bit later today.